config/valkyrie.yml

```diff
@@ -3,7 +3,7 @@ development:
   storage_adapter: disk
 
 test:
-  metadata_adapter: memory
+  metadata_adapter: postgres
```

Comment:
I'm fine with this for now — but I do think it's worth exploring this a bit locally and with the Data Mapper WG.  If different backends have different characteristics (which of course they do, or why bother?), do we need to be more explicit about those differences, and the tradeoffs of choosing one over another?